### PR TITLE
Update index.zh-hant.md

### PR DESCRIPTION
**Pull request by sss63232**
I've read the article many times. The original article and translation are great and helpful.🙏😀

But some words just make me confused. I've fixed some typo and polished some parts.

### CODE REVIEW

**Comment by sss63232** [comment]
Hi @eyeccc 
Could you help review this update🙏

File: src/pages/a-complete-guide-to-useeffect/index.zh-hant.md [hunk 1]
```diff
@@ -6,19 +6,19 @@ spoiler: Effect 是你資料流的一部分。
...
 
-但有時候，當你使用 `useEffect` 時，那些片段感覺不太契合。你有一種好像你遺漏了什麼的感覺。它看起來像是 class 的生命週期... 但它真的是嗎？你發現你問自己像是以下的某些問題：
+但有時候，當你使用 `useEffect` 時，某些程式碼片段感覺不太契合。你有一種好像你遺漏了什麼的感覺。它看起來像是 class 的生命週期... 但它真的是嗎？你發覺自己總是想著以下幾個問題：
 
 * 🤔 我要怎麼用 `useEffect` 複製 `componentDidMount` 的行為？
 * 🤔 我該怎麼正確的在 `useEffect` 裡拿到資料？`[]` 是什麼？
-* 🤔 我應該要把用到的函示指定成 effect 的依屬 (dependencies) 嗎？
-* 🤔 為什麼我有時候會進入重複拿資料的無窮迴圈？
+* 🤔 我應該要把用到的函式指定成 effect 的依屬 (dependencies) 嗎？
+* 🤔 為什麼我有時候會陷入重複拿取資料的無窮迴圈？
```
Comment (eyeccc) [inline src/pages/a-complete-guide-to-useeffect/index.zh-hant.md]: Maybe we should also modify L55 to make it consistent.

File: src/pages/a-complete-guide-to-useeffect/index.zh-hant.md [hunk 2]
```diff
@@ -6,19 +6,19 @@ spoiler: Effect 是你資料流的一部分。
...
 * 🤔 為什麼我有時候會在 effect 裡拿到舊的 state 或 prop 的值？
 
-當我剛開始使用 Hooks 的時候，我也對上述的問題感到困惑。甚至當我在寫初版的文件時，我沒有牢牢掌握一些細微的部分。在過程中我有好幾次的「啊哈」頓悟時刻，我想分享它們給你。**這樣的深入研究會使得這些問題的答案變得明顯。**
+當我剛開始使用 Hooks 的時候，我也對上述的問題感到困惑。甚至當我在寫初版的文件時，我沒有牢牢掌握一些細微的部分。在重新深入研究 `useEffect` 的過程中我有好幾次的「啊哈」頓悟時刻，我想把它們分享給你。**這樣的深入研究可以使你輕易理解上述問題的答案。**
 
-為了*看到*答案，我們需要先退回一步。這篇文章的目標不是給你一個條列式的清單，而是為了幫助你真的「深入理解」`useEffect`。這裡不會有很多需要學習的事情。事實上，我們會花大部分的時間*忘記*學習過的東西。
+為了*看到*答案，我們需要先退回一步。這篇文章的目標不是給你一個條列式的清單，而是為了幫助你真的「深入理解」`useEffect`。這裡不會 有很多需要學習的事情。事實上，我們會花大部分的時間*忘記*學習過的東西。
```
Comment (eyeccc) [inline src/pages/a-complete-guide-to-useeffect/index.zh-hant.md]: redundant whitespace in this line.

File: src/pages/a-complete-guide-to-useeffect/index.zh-hant.md (hunk 1 above)
Comment (sss63232) [reply src/pages/a-complete-guide-to-useeffect/index.zh-hant.md]: OK, I think so!

File: src/pages/a-complete-guide-to-useeffect/index.zh-hant.md (hunk 2 above)
Comment (sss63232) [reply src/pages/a-complete-guide-to-useeffect/index.zh-hant.md]: My fasult, thanks for the Eagle eye! 😆

**Pull request merged**